interfaces/builtin/greengrass_support.go

```diff
@@ -188,18 +355,29 @@ sethostname
 # Note that the lambda functions themselves run under a seccomp sandbox setup
 # by greengrassd.
 keyctl
+
+# device creation is necessary for mounting a squashfs file using loopback devices
+mknod
+mknodat
+
 `
 
 func init() {
+	// combine the greengrass-support iface code with the network-control code
+	// this is needed because plugging network-control normally by the greengrass snap
+	// will trigger a device cgroup to be enforced for greengrass which
+	// causes greengrass to fail, as greengrass needs to manage cgroups for it's
+	// containers itself
```

Comment:
This is no longer necessary, as we will now have the greengrass-support interface disable all udev tags and thus disable the device cgroup from ever being enabled. This will be implemented in a followup PR however.